gbpservice/nfp/configurator/drivers/firewall/vyos/vyos_fw_driver.py

```diff
@@ -701,3 +701,19 @@ def delete_firewall(self, context, firewall, host):
             self._print_exception('Failure', resp.status_code, url,
                                   'create', resp.content)
             return const.STATUS_ERROR
+
+    def configure_healthmonitor(self, context, kwargs):
+        """Overriding BaseDriver's configure_healthmonitor().
+           It does netcat to CONFIGURATION_SERVER_PORT  8888.
+           Configuration agent runs inside service vm.Once agent is up and
+           reachable, service vm is assumed to be active.
+           :param context - context
+           :param kwargs - kwargs coming from orchestrator
+
+           Returns: SUCCESS/FAILED
+
+        """
+        ip = kwargs.get('mgmt_ip')
+        port = str(const.CONFIGURATION_SERVER_PORT)
+        COMMAND = 'nc '+ip+' '+port+' -z'
```

Comment:
This will break the PEP8 and flake8 test. Please check. Deepak already ran these tests and fixed issues.